test/e2e/e2e_suite_test.go

```diff
@@ -210,8 +211,9 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 
 	if testTarget == VCSimTestTarget {
 		Byf("Creating a vcsim server")
-		err := vspherevcsim.Create(ctx, bootstrapClusterProxy.GetClient())
-		Expect(err).ToNot(HaveOccurred(), "Failed to create VCenterSimulator")
+		Eventually(func() error {
+			return vspherevcsim.Create(ctx, bootstrapClusterProxy.GetClient())
```

Comment:
Is this really retryable? I would expect the Create call to fail on the second try (because we don't ignore AlreadyExists).

So this only helps if the first try fails at the Create call with network issues

Reply:
That's correct, from what I observed, this call failed because controllers were not ready yet (and thus the webhook was not answering, which is a sort of network error).

Reply:
Got it!